.github/workflows/e2e-test.yml

```diff
@@ -13,6 +13,7 @@ on:
 
 env:
   GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  IMAGE_TRANSFORMER_IMG: "kserve/image-transformer:${{ github.sha }}"
```

Comment:
Can we set the env variable in python tests itself using fixtures or something ? This will make the local testing bit easier. As for the rest LGTM